modules/EmailMan/language/en_us.lang.php

```diff
@@ -132,7 +132,7 @@
     'LBL_PREPEND_TEST' => '[Test]: ',
     'LBL_SEND_ATTEMPTS' => 'Send Attempts',
     'LBL_OUTGOING_SECTION_HELP' => 'Configure the default outgoing mail server for sending email notifications, including workflow alerts.',
-    'LBL_ALLOW_DEFAULT_SELECTION' => 'Users may send as this account:',
+    'LBL_ALLOW_DEFAULT_SELECTION' => 'Users may send as this account\'s identity:',
```

Comment:
You shouldn't need to use an escape here, just use double quotes. See Quotations in the coding standards.

Reply:
Fixed.